### PR TITLE
Resize example

### PR DESCRIPTION
**Pull request by mjtalbot**
adding resize example, draft because i'm not sure we want to merge this. but would love to know what people make of it. 

for now we've gone with the resize observer throttled. would *love* to be able to fix this though... 

see video, or test it yourself, i've tried to add comments in for how to test this

https://user-images.githubusercontent.com/1216025/180247836-935b67a2-3106-49dd-a70a-505182242120.mov

### CODE REVIEW

File: examples/resize-test/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+# Resize issue: 
+
+update parameters in `utils.ts` to see this for yourself.
+
+Resizing from window.resize
+    - bottom animation moves slower than rest
+    - cannot deal with the animations container resizing, unless its linked to the window resizing
+Resizing from ResizeObserver
```
Comment (zplata) [inline examples/resize-test/README.md:24]: Yeah this one seems to be super buggy-looking when testing locally, and on that video

File: examples/resize-test/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+2. cd to `rive-wasm/js` & run `npm run dev` (keep this going as it will watch for changes)
+3. run `npm start`
+4. run `npm run build` for the rive-react project
+
+
+# Resize issue: 
+
+update parameters in `utils.ts` to see this for yourself.
+
+Resizing from window.resize
```
Comment (zplata) [inline examples/resize-test/README.md:21]: Running locally, this seemed a tad smoother than option 3 personally, but yeah you don't get the benefit of the last point.

**Review by zplata — COMMENTED** [review]
I think this change makes sense. Probably could reason about what value to delay by but also curious to hear @avivian thoughts on this.

Is option 2 what we're experiencing today because we do have the resize observer without throttle?

**Comment by avivian** [comment]
If the delay is set to 0, it's super smooth and doesn't have any of the flickering. I think maybe that's the best approach. Still would love to know exactly why this is breaking.

**Pull request closed without merging**